src/basic/architecture.h

```diff
@@ -43,6 +43,7 @@ enum {
         ARCHITECTURE_TILEGX,
         ARCHITECTURE_X86,
         ARCHITECTURE_X86_64,
+        ARCHITECTURE_LOONGARCH64,
```

Comment:
this was recently sorted properly, please keep this sorted

Reply:
Done.